### PR TITLE
Db 504 token reuse

### PR DESCRIPTION
**Pull request by jworcestBAH**
Tokens for password reset and registration are invalidated after process complete, and checked by unit tests.

### CODE REVIEW

File: dataactbroker/handlers/accountHandler.py [hunk 1]
```diff
@@ -387,7 +392,10 @@ def setNewPassword(self):
         user = self.interfaces.userDb.getUserByEmail(requestDict.getValue("user_email"))
         # Set new password
         self.interfaces.userDb.setPassword(user,requestDict.getValue("password"),self.bcrypt)
-
+        # Invalidate token
+        print("Deleting token: " + str(session["token"]))
```
Comment (bsweger) [inline dataactbroker/handlers/accountHandler.py]: Is this a leftover debug statement?

**Comment by bsweger** [comment]
@jworcestBAH Done reviewing. Looks good except for a possible debug statement lurking in there.

**Comment by jworcestBAH** [comment]
Good catch, removed the debug statement.

**Comment by bsweger** [comment]
Looks good--merging!

**Pull request merged**